www/docs/sources_destinations.md

```diff
@@ -1,9 +1,32 @@
 # Data sources and destinations
 
+Skip is designed to carefully track the computation and data dependencies of
+your application, so that outputs can be efficiently updated if (and only if!)
+any inputs they depend upon change.  However, when those inputs are from
```

Comment:
```suggestion
any inputs they depend on change.  However, when those inputs are from
```
since "upon" is more formal than "on" and the rest of our prose is not super formal, but your call